.gitignore

```diff
@@ -17,8 +17,13 @@ npm-debug.log
 dump.rdb
 manifest.json
 .cache-loader
+
+/data
+
+/src/__generated__
```

Comment:
I think it’s better to check these in. These are sometimes helpful during review, they speed up launch after pulling changes, and they will also contain the persisted query maps, for when we start using those in Force.